dashboard/app/controllers/lti/v1/integrations_controller.rb

```diff
@@ -0,0 +1,79 @@
+# frozen_string_literal: true
+
+require "queries/lti"
+require "services/lti"
+require "policies/lti"
+require 'metrics/events'
+
+module Lti
```

Comment:
Please add this acronym to the [inflections](https://github.com/code-dot-org/code-dot-org/blob/P20-1451-use-resourceful-routing-rules-for-lti/dashboard/config/initializers/inflections.rb)
```suggestion
module LTI
```

Reply:
Will Do!

Reply:
I will have to do this in a different PR because there are so many Lti references that I would need to change too many parts of the code unrelated to this PR.
We can create a separate task for this.